modules/openapi-generator/src/main/resources/aspnetcore/2.1/Startup.mustache

```diff
@@ -44,17 +44,13 @@ namespace {{packageName}}
         {
 {{#authMethods}}
 {{#isApiKey}}
-{{#-first}}
             services.AddTransient<IAuthorizationHandler, ApiKeyRequirementHandler>();
             services.AddAuthorization(authConfig =>
             {
-{{/-first}}            
                 authConfig.AddPolicy("{{name}}",
                     policyBuilder => policyBuilder
                         .AddRequirements(new ApiKeyRequirement(new[] { "my-secret-key" },"{{name}}")));
-{{#-last}}
             });
-{{/-last}}
```

Comment:
@phatcher thanks for the PR. I noticed these `-first`, `-last` tags are removed. Would it still work if the openapi spec contains multiple API key definitions?

Reply:
Not sure, I removed them as the code did not compile with a single API key, there were missing "}" from a number of files,  e.g. the PetStore example, is there some conditional code missing from the aspnetcore generator?.

Is there an example with multiple API key definitions so I can check what affect -first/-last have?

Reply:
OK. I think we can deal with it later as single API key is a lot more common.